test/cases/misc.zig

```diff
@@ -58,40 +59,40 @@ test "floating point primitive bit counts" {
     assert(f64.bit_count == 64);
 }
 
-test "@minValue and @maxValue" {
-    assert(@maxValue(u1) == 1);
-    assert(@maxValue(u8) == 255);
-    assert(@maxValue(u16) == 65535);
-    assert(@maxValue(u32) == 4294967295);
-    assert(@maxValue(u64) == 18446744073709551615);
+test "std.math.minValue and std.math.maxValue" {
```

Comment:
This should probably exist in `std.math`. I'm pretty sure these tests test language features.

Reply:
I thought so too, but I am glad that everyone agrees -- I will move it over to std.math